public/assets/js/pages/category.js

```diff
@@ -0,0 +1,10 @@
+document.addEventListener('DOMContentLoaded', function () {
+    $('#category').DataTable({
+        "language": {
+            "info": "Mostrando _START_ a _END_ de _TOTAL_ categorías",
+            "infoEmpty": "Mostrando 0 a 0 de 0 categorías",
+            "infoFiltered": "(filtrado de _MAX_ categorías totales)",
+            "lengthMenu": "Mostrar _MENU_ categorías",
+        }
+    });
```

Comment:
```suggestion
    $('#category').DataTable({
        "language": {
            "info": "Mostrando _START_ a _END_ de _TOTAL_ categorías",
            "infoEmpty": "Mostrando 0 a 0 de 0 categorías",
            "infoFiltered": "(filtrado de _MAX_ categorías totales)",
            "lengthMenu": "Mostrar _MENU_ categorías",
        },
         columnDefs: [
            { orderable: false, responsivePriority: 1, targets: -1}
         ]   
    });
```
Añade esto porfavor @FernandoJoachin para mejorar la tabla